controllers/users.js

```diff
@@ -4,6 +4,23 @@ const bcrypt = require("bcrypt")
 const User = require("../models/user")
 SALT_LENGTH = 12
 
+router.get("/:userId", async (req, res) => {
+	try {
```

Comment:
There's a missing gaurd phrase to protect the rotue:
`		if (!userProfile.isOwner(req.body.user)) {
			return res.status(403).json({
				error: "Oops! It doesn't look like that belongs to you!",
			})
		}
`